packages/sanity/src/form/studio/inputResolver/helpers.ts

```diff
@@ -29,8 +30,9 @@ export function getArrayFieldLevel(field: ArrayFieldProps): number {
   const {options} = field.schemaType
 
   const hasListOptions = (options?.list || [])?.length > 0
+  const isCustomArrayLayout = CUSTOM_ARRAY_LAYOUTS.includes(options?.layout || '')
```

Comment:
I'm wondering if we could find a better name than "custom" here - usually "custom" means "user defined". Given that we only handle a single known layout here, maybe we could just write it explicitly?  

```suggestion
const isTagsArray = options?.layout === 'tags'
```

Reply:
Good point, I'll rename it 👍